app/src/components/ui/NotFound/NotFound.module.scss

```diff
@@ -0,0 +1,17 @@
+@use 'styles/spacings';
+
+.container {
+  display: flex;
```

Comment:
I noticed that we are duplicating these lines, we could create a global placeholder and just extend it where we need it

%container-column {
  display: flex;
  flex-direction: column;
  justify-content: center;
}